### DISCUSSION TITLE
CMU 15-462 - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#360] [Announcements]
# CMU 15-462 - CS自学指南

CS自学指南

https://csdiy.wiki/%E8%AE%A1%E7%AE%97%E6%9C%BA%E5%9B%BE%E5%BD%A2%E5%AD%A6/15462/

**Comment by ruoyunbai** [comment]
重燃了我的学习之火！

**Comment by lhq5520** [comment]
let's go ，学就完事了

**Comment by KyleLu6** [comment]
我现在也要做加把劲骑士了

**Comment by PengyuJiang** [comment]
谢谢你🙏大家一起加油

**Comment by Erlend31** [comment]
为了我的北美梦！

**Comment by Mz-unitst** [comment]
寒假冲冲冲！csapp+net的lab!

**Comment by voania** [comment]
留爪

**Comment by lightcream** [comment]
我是24考研的学生，408相关内容看以上相关课程可以满足应试需求吗，评论区如果有前辈看到的话，求解答一下

**Reply by zrmin** [reply]
考研还是建议看王道，这些课程的目标和考研要拿高分的需求不一致

**Reply by lightcream** [reply]
了解了，谢谢学长！

**Comment by Mz-unitst** [comment]
可以，但是应试效果差。建议跟王道全程班，达到130的几率较大。




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年12月26日(星期一) 晚上9:13
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [PKUFlyingPig/cs-self-learning] CMU 15-462 - CS自学指南 (Discussion #360)





 
我是24考研的学生，408相关内容看以上相关课程可以满足应试需求吗，评论区如果有前辈看到的话，求解答一下

**Comment by wuyafei** [comment]
提供一个比较好的翻译版本：https://www.bilibili.com/video/BV15p4y1L7N2/?vd_source=2b9f6fcc0b53b5f5e1607a9e3e8c83bd

**Comment by xuzheng465** [comment]
求助如何在vscode debugger中启动Scotty3D...build文件只给了一个 Maekfile.js。难道要从里面扣配置来配置.vscode/launch.json吗？
